charts/enmasse/templates/Deployment-address-space-controller.yaml

```diff
@@ -0,0 +1,116 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+{{ include "enmasse.labels" . | indent 4 }}
+  name: {{ include "enmasse.fullname" . }}-address-space-controller
+spec:
+  replicas: {{ .Values.addressSpaceController.replicaCount }}
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "enmasse.fullname" . }}-address-space-controller
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  template:
```

Comment:
Would it worth to support adding of annotations here (e.g. for prom scrap config)?
(same for other deployments in this PR)